src/OrchardCore.Modules/Orchard.Users/Services/UserRoleRemovedEventHandler.cs

```diff
@@ -0,0 +1,26 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Orchard.Security;
+
+namespace Orchard.Users.Services
+{
+    public class UserRoleRemovedEventHandler : IRoleRemovedEventHandler
```

Comment:
I want to remove the event bus. We prefer to use specific interfaces in abstraction project. I think you can keep this interface, just remove anything that has to do with the even bus.